packages/core/test/predictableExec.test.ts

```diff
@@ -259,11 +259,11 @@ describe('predictableExec', () => {
       initial: 'active',
       context: ({ spawn }) => {
         const localId = ++invokeCounter;
-        actual.push(`start ${localId}`);
```

Comment:
as actors are created at a different time than they are started, I moved those pushes to the start's site to fix how the expectations were recorded in this file. Those tests were introduced in v4 where there was no distinction between those two times, so the previous version was OK there.